ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -29,13 +32,16 @@ public typealias MSPillButtonStyle = PillButtonStyle
 public enum PillButtonStyle: Int {
     case outline
     case filled
+    case gradient
```

Comment:
rather than another style, can we expose the button colors to be override by the client?

Reply:
reverted the change and exposed the background color overriding in pillbuttonbar.

---
In reply to: [465861540](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r465861540) [](ancestors = 465861540)

Reply:
in new iteration, instead of using a new style, we are taking pill background color as optional value in pill bar's initializer

---
In reply to: [465887853](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r465887853) [](ancestors = 465887853)